torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -8,10 +8,15 @@
 #include <torch/csrc/autograd/function_hook.h>
 #include <torch/csrc/autograd/functions/accumulate_grad.h>
 #include <torch/csrc/autograd/profiler.h>
+#include <torch/csrc/distributed/c10d/comm.h>
 #include <torch/csrc/utils/hash.h>
 #include <torch/csrc/utils/memory.h>
 
 namespace c10d {
+
+constexpr int DEFAULT_FIRST_BUCKET_BYTES = int(1024 * 1024);
+constexpr int DEFAULT_BROADCAST_BUCKET_BYTES = int(250 * 1024 * 1024);
```

Comment:
what's the reason for using 10X broadcast buckets byte sizes compared to default allreduce buckets?

Reply:
distributed.py uses 10x broadcast buckets byte sizes in default, my understanding is because broadcast is cheaper than all reduce, so it should be fast enough to broadcast much larger buckets